k8s/aat/common/ccd/test-stubs-service.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-test-stubs-service
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccd-test-stubs-service
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccd-test-stubs-service
+    version: 1.0.2
+  values:
+    java:
+      replicas: 2
```

Comment:
i guess, one replica is fine for this service. So total of two instances (both clusters)